# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/tests/mochitests/pc.js
# Commit: 9d8f6e354d8c
# Full Hash: 9d8f6e354d8c332bab7e433252077ec0e1fa8f04
# Author: John Lin <jolin@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 1
# Description:
#   Bug 1654112 - remove MediaCodecVP8Codec. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D115531
# ==============================================================================

diff -r 7675ead797c8 -r 9d8f6e354d8c dom/media/webrtc/tests/mochitests/pc.js
--- a/dom/media/webrtc/tests/mochitests/pc.js	Fri Oct 08 15:57:53 2021 +0200
+++ b/dom/media/webrtc/tests/mochitests/pc.js	Thu Oct 07 12:11:40 2021 -0700
@@ -2549,29 +2549,11 @@
     "resource://gre/modules/AppConstants.jsm",
     {}
   );
-  let isNightly = AppConstants.NIGHTLY_BUILD;
-  let isAndroid = AppConstants.platform == "android";
 
   await scriptsReady;
   await runTestWhenReady(async options => {
     await startNetworkAndTest();
     await setupIceServerConfig(fixtureOptions.useIceServer);
-
-    // currently we set android hardware encoder default enabled in nightly.
-    // But before QA approves the quality, we want to ensure the legacy
-    // encoder is working fine.
-    if (isNightly && isAndroid) {
-      let value = Math.random() >= 0.5;
-      await SpecialPowers.pushPrefEnv({
-        set: [
-          ["media.navigator.hardware.vp8_encode.acceleration_enabled", value],
-          [
-            "media.navigator.hardware.vp8_encode.acceleration_remote_enabled",
-            value,
-          ],
-        ],
-      });
-    }
     await testFunction(options);
     await networkTestFinished();
   });